contracts/PetPark.sol

```diff
@@ -1,7 +1,108 @@
 //SPDX-License-Identifier: Unlicense
 pragma solidity ^0.8.0;
 
-
 contract PetPark {
+    address owner;
+
+    mapping(AnimalType => uint256) private shelter;
+    mapping(address => Person) private borrowers;
+
+    struct Person {
+        uint256 age;
+        Gender gender;
+        AnimalType animal;
+        bool initialized;
+    }
+
+    event Added(uint256 AnimalType, uint256 Count);
```

Comment:
Here `animalType` can be of enum AnimalType rather than uint256

Reply:
yep, typo.